lib/prometheus_exporter/middleware.rb

```diff
@@ -41,7 +42,8 @@ def call(env)
       type: "web",
       timings: info,
       queue_time: queue_time,
-      default_labels: default_labels(env, result)
+      default_labels: default_labels(env, result),
+      options: { mode: @mode }
```

Comment:
I wonder if we can avoid sending this piece of text with every metric? Maybe we use a different way of initializing that sets up the collector so it know it is getting historams vs summaries?